polars/polars-sql/src/sql_expr.rs

```diff
@@ -374,5 +374,11 @@ pub(super) fn process_join_constraint(
             _ => {}
         }
     }
+    if let JoinConstraint::Using(idents) = constraint {
+        if let 1.. = idents.len() {
```

Comment:
What do you want to test here? If it is 1 or bigger than one?

Explicitly writing that test is more clear than pattern matching a range.

`if idents.len() == 1 {`

Reply:
Ahh... I originally had an extra branch, then clippy simplified it and I didn't finish the job, hah. Fixed! 👍